manifests/init.pp

```diff
@@ -0,0 +1,166 @@
+  Integer[0,1]                $copy_log_on_error                 = 0,
+  Optional[String[1]]         $dbdir                             = undef,
+  Optional[Array[String[1]]]  $disable_tests                     = undef,
+  Optional[Array[String[1]]]  $empty_logfiles                    = undef,
+  Array[String[1]]  $enable_tests                                = ['ALL'],
+  Optional[String[1]]         $epoch_date_cmd                    = undef,
+  Optional[Array[String[1]]]  $exclude_user_fileprop_files_dirs  = undef,
+  Optional[Array[String[1]]]  $existwhitelist                    = undef,
+  Integer[0,1]                $globstar                          = 0,
+  String[1]                   $hash_cmd                          = 'SHA256',
+  Integer[1]                  $hash_fld_idx                      = 1,
+  Optional[Array[String[1]]]  $ignore_prelink_dep_err            = undef,
+  Integer[0,1]                $immutable_set                     = 0,
+  Optional[Array[String[1]]]  $immutwhitelist                    = undef,
+  Optional[Array[String[1]]]  $inetd_allowed_svc                 = undef,
+  Optional[String[1]]         $inetd_conf_path                   = undef,
+  Optional[String[1]]         $installdir                        = undef,
+  Integer[1]                  $ipc_seg_size                      = 1048576,
+  String[1]                   $language                          = 'en',
+  String[1]                   $logfile                           = '/var/log/rkhunter/rkhunter.log',
+  Optional[String[1]]         $mail_on_warning                   = undef,
+  String[1]                   $mail_cmd                          = 'mail -s "[rkhunter] Warnings found for ${HOST_NAME}"', # lint:ignore:single_quote_string_with_variables
+  Integer[0,2]                $mirrors_mode                      = 0,
+  Optional[Array[String[1]]]  $missing_logfiles                  = undef,
+  Optional[String[1]]         $modules_dir                       = undef,
+  Optional[String[1]]         $os_version_file                   = undef,
+  Optional[String[1]]         $password_file                     = undef, #should keep? suggestion: /etc/shadow
+  Integer[0,1]                $phalanx2_dirtest                  = 0,
+  Optional[Array[String[1]]]  $pkgmgr_no_vrfy                    = undef,
+  String[1]                   $pkgmgr                            = 'RPM',
+  Optional[Array[String[1]]]  $port_path_whitelist               = undef,
+  Optional[Array[String[1]]]  $port_whitelist                    = undef,
+  Optional[Array[String[1]]]  $pwdless_accounts                  = undef,
+  Optional[String[1]]         $readlink_cmd                      = undef,
+  Integer[0,1]                $rotate_mirrors                    = 1,
+  Optional[Array[String[1]]]  $rtkt_dir_whitelist                = undef,
+  Optional[Array[String[1]]]  $rtkt_file_whitelist               = undef,
+  Enum['THOROUGH','LAZY']     $scan_mode_dev                     = 'THOROUGH',
```

Comment:
This is OK to leave this way even though it is the default. It lets the user know some important contextual information.